LayoutTests/fast/svg/path-NaN-stroke-bounding-box-crash.html

```diff
@@ -0,0 +1,19 @@
+<p>This test passes if it doesn't crash.</p>
+<script>
+if (testRunner)
+    testRunner.dumpAsText();
+function jsfuzzer() {
```

Comment:
Can we make this less like a fuzzer test case? Give this function a normal name, give the variables reasonable names.